src/pipelines/orchestrator.py

```
@@ -994,6 +1053,49 @@ def _hydrate_openai_config(self) -> Optional[OpenAIProviderConfig]:
+            merged.update(raw)
+        elif isinstance(raw, OpenAIProviderConfig):
+            merged.update(raw.model_dump())
+
+        if not merged:
+            for _, cfg in providers.items():
+                if not isinstance(cfg, dict):
+                    continue
+                base = str(cfg.get("chat_base_url") or cfg.get("base_url") or "").strip()
+                try:
+                    host = (urlparse(base).hostname or "").lower()
+                except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)